raster/loader/raster2pgsql.c

```diff
@@ -2522,7 +2522,7 @@ main(int argc, char **argv) {
 			elements = NULL;
 			n = 0;
 
-			for (j = 0; j < config->overview_count; j++) {
+			for (j = 0; j < (uint32_t)config->overview_count; j++) {
```

Comment:
can overview_count be unsigned on declaration?

Reply:
Switching overview_count and rt_file_count both to uint32_t